_data/changelogs/accessibility.yml

```diff
@@ -1,9 +1,8 @@
-title: Accessibility
-pageURL: accessibility
+pageName: Accessibility
+pageType: Guidance
```

Comment:
**Optional** we could simplify to `name/type`, but I can see the added clarity the page keyword brings

Reply:
I support it. There were initially more items up top so I liked the added clarity, but it doesn't seem necessary anymore. Updated in [51a9384](https://github.com/uswds/uswds-site/pull/1789/commits/51a9384c2feb9d225a743dbdac1d06d5d55053d7)